includes/languages/english/extra_definitions/ask_a_question_defines.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+define('ASK_A_QUESTION', 'Ask a question about this product');
+define('BUTTON_IMAGE_ASK_A_QUESTION','button_ask_a_question.gif');
+define('BUTTON_ASK_A_QUESTION_ALT', 'Ask a Question');
```

Comment:
I can't decide whether to leave this standalone or merge it into core files.
Perhaps at least add a `// @TODO` header note to this file, to consider the merge at a future point.